recipes/protozfits/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "protozfits" %}
+{% set version = "2.0.1.post1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: a3864c6097196208fca15b17eb49241675b3811bd6d1ffec206ce0e3bba8f89f
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [win]
+  skip: true  # [py<36]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+  host:
+    - pip
+    - python
+    - protobuf
+    - libprotobuf
+    - setuptools
+    - zeromq
+    - zlib
+  run:
+    - astropy
+    - numpy
+    - protobuf
```

Comment:
I suspect this isn't needed at runtime
```suggestion
```

Reply:
The `google.protobuf` package is needed at runtime.

Reply:
That should be the `protobuf` package, right?

Reply:
Yes, it is. So protobuf in the runtime dependencies is correct.

Reply:
Indeed, sorry I didn't notice `protobuf` is the Python bindings 👍